meetings/2022-03-09.md

```diff
@@ -0,0 +1,134 @@
+* Matthieu: Excited about possible Python implementation. If you have any question about ACP spec don't hesitate to ping me on gitter.
+
+---
+
+## Topics
+
+### Actions from last week
+
+* Matthieu to create an issue on the authorization panel to make the `acp:client` property closed by default.
+* Matthieu to create an issue on the authorization panel to make all ACP matcher properties closed by default (and potentially raise it as a general authz issue affecting both the ACP and ACL ontologies).
+  * Matthieu: will do it after this meeting.
+* Elf to create an issue about the requirement to not disclose a trusted issuer publicly in the Solid-OIDC panel and raise it in the panel on Monday the 7th.
+  * https://github.com/solid/solid-oidc/issues/92
+* Elf to write an issue for a VC use case showing the responsibility of the UMA AS.
+  * https://github.com/solid/authorization-panel/discussions/290
+
+
+### New person perspective
+
+* Elf: What does the current state of WAC/ACP seems for someone
+* Kushal: I only learned about semantic data late 2015. When I started reading the specs, I found it a bit overwhelmed and had to go back to the fundamentals. It's a little bit confusing to get to understand the different parts of the Solid ecosystem. As a beginner even though I'm writing code for many years, it's difficult getting into it. The examples really help.
+* Elf: Some specs have primers (for example Solid OIDC). We're trying as editors to make them available.
+* Matthieu: There is no primer for ACP, I'll try to write one up.
+* Kushal: Personally I try to write my questions in [my blog](https://kushaldas.in/posts/solid-project-webid-and-privacy.html).
+* ...: I was wondering why the Authorization server and the resource server have to be separate is one of my questions.
+* ...: This comes more from an implementation point of view. Generally, people don't have to separate it. But I'll come back to this after reading more about it.
+* Elf: It's regular OAuth implementation to separate authz and resource server. Also, while this is described separately, some implementations can choose to combine those roles.
+
+
+### Responsibility of the UMA AS and ACP engine in VC checking
+
+URL: https://github.com/solid/authorization-panel/discussions/290
+
+* Elf: It's a rough example relying on an existing use case (see [discussion on the authorization repo](https://github.com/solid/authorization-panel/discussions/290)).
+* Wouter: Very interesting. Why would the UMA server need to access the ACL? Couldn't the VC Type be passed along by the server in the ticket (I'd like to keep things decoupled).
+* Elf: I think that the AS knows the policy, from the client's perspective, it doesn't matter. I think it's between the RS and AS that the discovery happens. Some kind of a JWT might be used. It would be nice to decouple AS and RS.
```

Comment:
```suggestion
* Elf: I think that the AS knows the policy; from the client's perspective, it doesn't matter. I think it's between the RS and AS that the discovery happens. Some kind of a JWT might be used. It would be nice to decouple AS and RS.
```